test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -782,10 +753,17 @@ class test_ucp_peer_failure_rndv_abort : public test_ucp_peer_failure {
         switch (mode) {
         case rndv_mode::rndv_get:
             m_proto_name    = "rndv/get/zcopy";
+            m_proto_stage   = UCP_PROTO_STAGE_START;
```

Comment:
maybe rename m_proto_stage to m_proto_xfer_stage, document that it is the protocol stage during which data transfer happens, and here use UCP_PROTO_STAGE_START instead of UCP_PROTO_STAGE_START